sdk/python/foundation-models/system/inference/image-text-embeddings/text-to-image-retreival.ipynb

```diff
@@ -0,0 +1,586 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Text-to-Image Retrieval using Online Endpoints and Indexes in Azure AI Search\n",
+    "\n",
+    "This sample shows how to perform vector search with a Azure AI Search Index and a deployed `embeddings` type model.\n",
```

Comment:
[nit] "sample" -> "example"
[typo] "a Azure AI Search Index" -> "an Azure AI Search Index"
Also, "vector search" is general and it would be better to be specific here, eg "text-to-image search".